CJSpohn/ideabox-boilerplate
# Feature/iteration 0

Is this a fix or a feature?
Feature

What is the change/fix?
Contains all the HTML and CSS styling to match the comp

Where should the reviewer start?
At the beginning of index.html and styles.css

How should this be tested?
Open in the browser and check that the page looks like the given comp.

Screenshots(if appropriate)
<img width="1100" alt="Screen Shot 2020-10-28 at 4 19 58 PM" src="https://user-images.githubusercontent.com/69563078/97502786-7a436700-1939-11eb-9d6b-afce9f490620.png">
